proto/stream_plan.proto

```diff
@@ -113,10 +113,9 @@ message HashJoinNode {
 message DynamicFilterNode {
   uint32 left_key = 1;
   // Must be one of <, <=, >, >=
-  expr.ExprNode condition = 3;
+  expr.ExprNode condition = 2;
   // Used for internal table states.
-  // catalog.Table left_table = 4;
-  // uint32 dist_key_l = 8;
+  catalog.Table left_table = 3;
```

Comment:
Will there be right_table?

Reply:
I think so, it may be a single row/value

Reply:
Not sure what's PK for simple agg? (Is it empty?). Basically we want a state table with just table id and col id as value prefix

Reply:
Yes, we are allowed to insert a Row(vec![]).

Reply:
Ok. I think this is achieved if dist_key and pk_indices are both empty.